_includes/04-lists.md

```diff
@@ -0,0 +1 @@
+Replace this with a list of your favorite things.
```

Comment:
List of my favorite books:
* _1984_ by **George Orwell**
* _UBIK_ by **Phillip K. Dick**
* _Crime and Punishment_ by **Fyodor Dostoevsky**